ps_sharebuttons.php

```diff
@@ -176,7 +177,7 @@ public function getWidgetVariables($hookName, array $params)
             $social_share_links['facebook'] = [
                 'label' => $this->trans('Share', [], 'Modules.Sharebuttons.Shop'),
                 'class' => 'facebook',
-                'url' => 'https://www.facebook.com/sharer.php?u='.$sharing_url,
+                'url' => 'http://www.facebook.com/sharer.php?u='.$sharing_url.'&picture='.$sharing_img.'&title='.$sharing_name.'&description='.$sharing_desc,
```

Comment:
I think we can keep with `https` here.
```suggestion
                'url' => 'https://www.facebook.com/sharer.php?u='.$sharing_url.'&picture='.$sharing_img.'&title='.$sharing_name.'&description='.$sharing_desc,
```